recipes/r-mrgsolve/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "r-mrgsolve" %}
+{% set version = "0.11.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: 
+    - {{ cran_mirror }}/src/contrib/mrgsolve_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/mrgsolve/mrgsolve_{{ version }}.tar.gz
+  sha256: a7b1dd4c8c8f985372ece37679dd644c30f7fa6aa16dba265120982a925cf9c1
+
+build:
+  number: 0
+  noarch: generic
+  rpaths:
+      - lib/R/lib/
+      - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+  host:
+    - r-base
+    - r-magrittr
+    - r-tidyselect
+    - r-tibble
+    - r-rlang
+    - r-rcpp
+    - r-dplyr
+  run:
+    - r-base
+    - r-magrittr
+    - r-tidyselect
+    - r-tibble
+    - r-rlang
+    - r-rcpp
+    - r-dplyr
+
+test:
+  commands:
+    - $R -e "library('mrgsolve')"           # [not win]
+    - "\"%R%\" -e \"library('mrgsolve')\""  # [win]
+
+
+about:
+  home: https://github.com/metrumresearchgroup/mrgsolve
+  license: GPL-2.0-or-later
+  summary: Simulate from ODE-based population PK/PD and QSP models in R
+  license_family: GPL
+  license_file: mrgsolve_licence.txt
```

No need to contain an extra file here... r-base already provides most of the licenses used by cran packages.
```suggestion
  license_file:
    - {{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2
```